src/bootstrap-table.js

```diff
@@ -394,6 +394,10 @@
 
         customSort: $.noop,
 
+        ignoreClickToSelectOn: function(element) {
+            return $.inArray(element.tagName, [ "A", "BUTTON" ];
+        },
```

Comment:
```
ignoreClickToSelectOn: function (element) {
    return $.inArray(element.tagName, ['A', 'BUTTON']);
},
```

Reply:
Whoops, sorry for the typo, I didn't actually test it out. I've updated the PR (along with the documentation), and actually tested it out this time. Things should be working now!